templates/enketo/deployment.yaml

```diff
@@ -28,8 +28,8 @@ spec:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       containers:
         - name: {{ .Chart.Name }}
-          # Override to avoid create config script, since we use env vars
-          command: ["node", "app.js"]
+          # Avoid upstream create config script and support proper SIGTERM handling
+          command: ["pm2-runtime", "app.js"]
```

Comment:
let's ditch this since the upstream PR is merged
https://github.com/enketo/enketo-express/pull/533